shared/src/utilities/JoiValidationDecorator.js

```diff
@@ -162,6 +162,7 @@ exports.joiValidationDecorator = function (
     });
 
     if (error) {
+      console.log('entity with error-------------', this);
```

Comment:
Probably want to remove this?

Reply:
I left it there intentionally for troubleshooting errors in migrations. It will only print out when `validateForMigration` is called and if there is an error.